quickstart/README.md

```diff
@@ -7,17 +7,32 @@
 
 ## Lab
 
+The container config of [Lobsters](https://github.com/jcs/lobsters) is
+[here](lobsters/). It is a test configuration with a local sqlite db
```

Comment:
consider adding a note that we've built the image and pushed it to gcr.io/google-samples/lobsters